app/recruitment/tests.py

```
@@ -0,0 +1,280 @@
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)